src/styles.scss

```diff
@@ -852,15 +866,15 @@ $green-font-color: #19B028;
 }
 
 .change-account-selector__hover:hover {
-  background-color: var(--secondary);
+	border: 1px solid rgba(101, 119, 134, 0.55);
```

Comment:
I would use one of the theme colors here instead of the rgba value so that it adjusts appropriately when themes are changed.